jest.config.js

```diff
@@ -81,6 +83,7 @@ module.exports = {
   globals: {
     DOCS_MODE: false,
     PREVIEW_URL: undefined,
+    SNAPSHOT_OS: os.platform() === 'win32' ? 'windows' : 'posix',
```

Comment:
By adding this boolean as a testing global, it makes it easier to consistently differentiate the os in our tests, mostly useful for snapshots, when combined with `toMatchSpecificSnapshot`.